doc/source/user/usage.rst

```diff
@@ -251,6 +251,15 @@ Similar to the SAS executable, the utilities are also OS dependent. To execute
 SAS, the :meth:`~watts.PluginSAS` instance is called directly in the same way as
 other plugins.
 
+WATTS also supports the "INCLUDE" directive of SAS that allows for multiple input
+files. Users simply need to add "INCLUDE extra_input_file_names" to the template
+and specify the names of the extra input files as a string to the
+"extra_template_inputs" argument when calling the :class:`~watts.PluginSAS` class::
+
+    sas_plugin = watts.PluginSAS('sas_template', show_stdout=True, extra_template_inputs=['extra_input_file_names'])
+
+An example has been provided to demonstrate this capbility.
+
```

Comment:
`An example has been provided to demonstrate this capbility` - fix typo (capability) and provide link to this new example.